### ISSUE TITLE
[detection / classification] Train upcoming detection architecture & orientation models

### ISSUE THREAD

**Issue by felixdittrich92** [mindee/doctr#1459]
### 🚀 The feature

FAST - Detection model

PyTorch:

- [ ] `fast_tiny`
- [ ] `fast_small`
- [ ] `fast_base`

TF:

- [ ] `fast_tiny`
- [ ] `fast_small`
- [ ] `fast_base`

Orientation models:

Train-Skripts:
https://github.com/mindee/doctr/blob/main/references/classification/train_pytorch_orientation.py
https://github.com/mindee/doctr/blob/main/references/classification/train_tensorflow_orientation.py

TF & PT Crop: (with the recognition dataset)

- [ ] `mobilenet_v3_small_orientation`

TF & PT Page: (with the detection dataset)

- [ ] `mobilenet_v3_small_page_orientation`

### Motivation, pitch

x

**Comment by SkaarFacee** [comment]
Is anyone working on this ? I would love to contribute in any way possible. This is something I look forward to learn as well :)

**Comment by felixdittrich92** [comment]
Hi @SkaarFacee :) 
This issue can only be solved by @odulcy-mindee because he is the only one with access to the machines an mindee internal datasets :)

**Comment by SkaarFacee** [comment]
Oh understood. Is there any other similar issue I can work on instead? If not can you suggest some issues ?

**Comment by felixdittrich92** [comment]
Hey sure :)

https://github.com/mindee/doctr/issues/1375 Should be super easy and fast to solve :)

Or if you want to work on something bigger and more deep learning specific: 
https://github.com/mindee/doctr/issues/1411

**Comment by SkaarFacee** [comment]
Hey, I'll take a look at them. Thanks again

**Comment by felixdittrich92** [comment]
> Hi @SkaarFacee and @felixdittrich92, could you please share what is the status here? I checked "fast" detection models and it seems that they are not trained yet. Is there any chance you will add them in a nearly future?

Hey @TomekPro yep we finished some fixes and experiments today and will spin up the final runs now so we will publish the checkpoints one by one in the next weeks (1 model takes ~ 1week)

**Comment by TomekPro** [comment]
Great, thanks! Those will be on pytorch or TF?

**Comment by felixdittrich92** [comment]
> Great, thanks! Those will be on pytorch or TF?

We start with pytorch first :)

**Comment by felixdittrich92** [comment]
> @felixdittrich92 any update on this? Thanks!

Yep after crashing two machines it is running now i think we can publish on monday the first checkpoint 😅

**Comment by odulcy-mindee** [comment]
Yeah, we got a bit (too) unlucky this time :sweat_smile: Sorry for the delay, model checkpoint will be released soon

**Comment by felixdittrich92** [comment]
@TomekPro First model available (including reparameterization by default - if used with the `ocr_predictor(det_arch="fast_tiny",..)`   :)
